spec/integration/retryable_errors_spec.rb

```diff
@@ -59,41 +45,223 @@
       end
     end
 
-    let(:perform_read) do
-      client.cluster.servers_list.each do |server|
-        server.monitor.stop!
+    shared_context 'read operation' do
+      let(:fail_point_command) do
+        {
+          configureFailPoint: 'failCommand',
+          mode: {times: 1},
+          data: {
+            failCommands: ['find'],
+            errorCode: 11600,
+          },
+        }
       end
 
-      ClusterTools.instance.direct_client_for_each_server do |client|
+      let(:set_fail_point) do
+        client.cluster.servers_list.each do |server|
+          server.monitor.stop!
+        end
+
+        ClusterTools.instance.direct_client_for_each_server do |client|
+          client.use(:admin).database.command(fail_point_command)
+        end
+      end
+
+      let(:operation_exception) do
+        set_fail_point
+
+        begin
+          collection.find(a: 1).to_a
+        rescue Mongo::Error::OperationFailure => exception
+        else
+          fail('Expected operation to fail')
+        end
+
+        puts exception.message
+
+        exception
+      end
+
+      let(:events) do
+        EventSubscriber.command_started_events('find')
+      end
+
+      it 'sends reads to different servers' do
+      end
```

Comment:
Is this example meant to be empty?

Reply:
This is in fact handled elsewhere, thank you for noticing.